docs/userguide/containers/dockervolumes.md

```diff
@@ -270,6 +270,15 @@ Then un-tar the backup file in the new container's data volume.
 You can use the techniques above to automate backup, migration and
 restore testing using your preferred tools.
 
+## Removing Volumes
+
+Docker Volumes are designed to persist even after a container is deleted. However,
+they can be removed by using the `--rm` flag. For example with,
+
+    $ docker run --rm -v /foo -v awesome:/bar busybox top,
+
+the volume for `/foo` will be removed, but the volume for `/bar` will not.
+
```

Comment:
Been thinking how we should describe this. Perhaps something like;

    Docker volumes are designed to persist even after a container is deleted.
    However, if a container is started with the `--rm` flag, then any "anonymous"
    volume attached to the container is deleted when the container exits (and
    removed).

    For example:

    ```bash
    $ docker run --rm -v /foo -v awesome:/bar busybox top,
    ```

    In the example above, the volume for `/foo` is an "anonymous" volume (does not
    have a name) and is cleaned up after the container exits. However, the `awesome`
    volume will *not* be removed, because it was given a name ("awesome").

@cpuguy83 wdyt?